_posts/2020-04-17-snowplow-amp-tracker-1-0-1-released/index.md

```diff
@@ -0,0 +1,186 @@
+
+<h3 id="amp-id-entity"> 2.1 AMP ID Entity</h3>
+
+The AMP ID entity is a new feature to Snowplow tracking, with a specific purpose of making it easier to identify users on the AMP platform, and across AMP and non-AMP pages. By default, the AMP ID entity will be attached to all events, against the [AMP ID schema][amp-id-schema].
+
+This will contain the ampClientId (the consistent identifier for users on the AMP platform), the `user_id` if set, and the `domain_userid`, if provided by the Javascript tracker's cross-domain linker. (The AMP tracker does not generate a `domain_userid`).
+
+The `domain_userid` can be made available to the AMP tracker when a user moves from a javascript tracker page to an AMP page, by enabling the javascript tracker's [cross-domain tracking][cd-linker] feature. The javascript tracker will attach the `domain_userid` to the querystring, and the AMP tracker will automatically retrieve it and attempt to retain it across pages.
+
+Note that while the AMP tracker attempts to retain the domain userid across pages and sessions, the AMP platform does not offer any means to guarantee that this can be done in all cases - so the user identification strategy here is based on having the value attached to at least one event, rather than all events.
+
+<h3 id="amp-linker"> 2.2 AMP Linker</h3>
+
+The AMP tracker now offers the ability to attach the AMP client ID to the querystring, in order to identify users moving from an AMP page to a non-AMP page. This is enabled by ensuring that the AMP linker is enabled for any destination domains required:
+
+{% highlight html %}
+...
+"linkers": {
+  "enabled": true,
+  "proxyOnly": false,
+  "destinationDomains": ["ampdomain"]
+},
+...
+{% endhighlight %}
+
+This will add a querystring parameter ‘linker=’ to the destination url, which contains the amp_id value, base-64 encoded.
+
+
+<h3 id="modeling-users"> 2.3 Modeling Considerations</h3>
+
+The value attached to the querystring by the AMP linker will look something like this: `?linker=1*1c1wx43*amp_id*amp-a1b23cDEfGhIjkl4mnoPqr`. To extract the AMP ID, this must be parsed, and the value immediately following the `amp_id*` string must be extracted.
+
+This value will only be present for the first page the user lands on after leaving the AMP page.
+
+If a `domain_userid` is found by the AMP tracker, it is not guaranteed to be attached to every event. Therefore, a good strategy for modeling user identification on both sides is to create a mapping table of domain_userid to amp-id, and join this to the rest of the data to attribute users.
+
+
+<h2 id="custom-events">3. Custom Events</h2>
+
```

Comment:
Would it make sense to give a bit of background on why custom events and entities are a useful feature/what it enables, or is it assumed users would already be familiar with this?

Reply:
As it's a release post for a tracker, normally it would assume a certain amount of knowledge and focus on the features of the product at hand. 

The audience in mind is normally someone who is already using this specific tracker and wants to upgrade, or someone who already knows their tracking requirements and wants to implement. I would expect that by that point they're already up to speed on the basics.

I've added a link to the section in the docs that talks about tracking custom data in a general sense.